localization/uk_UA.lua

```diff
@@ -1171,7 +1171,7 @@
                 }
             },
             j_flash = {
-                name = "Флеш-карта",
+                name = "Флаш-карта",
```

Comment:
```suggestion
                name = "Флеш-карта",
```

тут залишаю Флеш-карту, тому що це стосуються ось цього
https://en.wikipedia.org/wiki/Flashcard
https://vseosvita.ua/news/flesh-kartky-efektyvnyi-sposib-zapamiatovuvannia-kliuchovoi-informatsii-16394.html